pkg/storage/tsdb/config.go

```diff
@@ -0,0 +1,48 @@
+package tsdb
+
+import (
+	"errors"
+	"flag"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/storage/tsdb/backend/s3"
+)
+
+const (
+	// BackendS3 is the const for the s3 backend for tsdb long-term retention
+	BackendS3 = "s3"
+)
+
+// Config holds the config information for TSDB storage
+type Config struct {
+	Dir          string        `yaml:"dir"`
+	SyncDir      string        `yaml:"sync-dir"`
```

Comment:
`sync-dir` > `sync_dir` please (by convention, all yaml is `_` separated, while CLI flags are `-` separated)

Reply:
Fixed